src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -585,6 +586,56 @@ public object[] Geometry()
             return converted.ToArray();
         }
 
+        /// <summary>
+        /// Returns all geometry associated with an element. Ignores transforms when used with linked elements.
+        /// <param name="detailLevel">Detail level</param>
+        /// <returns>List of geometry from the element</returns>
+        /// </summary>
+        public object[] GetGeometry(string detailLevel = "Medium")
```

Comment:
doesn't an enum make more sense here? What about localized envs? How will users know what strings to use etc? I think there are other examples in the repo - maybe?

Reply:
It actually works with the current `Detail Level` dropdown node in Dynamo which yields a string output. It looks like this:

![image](https://user-images.githubusercontent.com/5354594/235144437-073be845-311b-4976-9c80-361aa8047d01.png)

The output of the `Detail Level` node is not language dependent, so it made sense to do it this way. Let me know if that still bothers you though!